packages/igx-templates/igx-ts/dock-manager/index.ts

```diff
@@ -0,0 +1,17 @@
+
+import { BaseComponent } from "@igniteui/cli-core";
+
+class IgxDockManagerComponent extends BaseComponent {
+	/**
+	 *
+	 */
+	constructor() {
+		super(__dirname);
+		this.name  = "Dock Manager";
+		this.group = "Layouts";
+		this.description = `provides means to manage the layout of your application through panes,
```

Comment:
Not sure what the extra new lines and indent are in the string, but if it were used I'd assume it won't come out nice